_build_cfg.yml

```diff
@@ -210,6 +208,8 @@ Topics:
     File: syncing_groups_with_ldap
   - Name: Available Ports
     File: available_ports
+  - Name: Aggregating Logs with the EFK Stack
```

Comment:
Actually fluentd *does* feed service/platform logs into the system as well. I think taking over the old "aggregate_logging" filename would be fine.